### ISSUE TITLE
Changing content - unnecessarily live region per 4.1.3 failure example

### ISSUE THREAD

**Issue by cliffbert** [Section508Coordinators/ICTTestingBaseline#229]
[Understanding 4.1.3](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html#examples) has this TBD failure example: Using role="alert" or aria-live="assertive" on content which is not important and time-sensitive (future link)

I don't see where this is addressed in current baseline or test process - evaluating **unnecessarily assertive** live region (or other method).

**Comment by kengdoj** [comment]
SC 4.1.3 was added by WCAG 2.1, and is not part of 2.0, so it's not referenced in Revised Section 508.

**Comment by cliffbert** [comment]
True, but it seems like something they want to address, and it is probably something that we will run across in testing.

**Comment by drewnielson** [comment]
Agreed that it will come up in testing. Even though it is only new to WCAG
2.1 we might still want to address is somehow. I still have the same
reservations as I did when initially discussed it: I just think it is
really difficult to instruct testers about how to evaluate whether
something should or should not be intrusive (e.g., using role="alert"
(where the default aria-live is assertive) vs using role="marquee" (where
the default aria-live is off)). It just seems like there is way too much
subjectivity. We might still decide to take a pass until the next 508
refresh, but could still be helpful to think and talk through it.

**Comment by cliffbert** [comment]
Yes, we may want to come up with a Colin Robinson test, some kind of energy drainage measure for uber-chatty live regions.

**Comment by drewnielson** [comment]
bwahahaha! Binged all of the episodes from that show a couple weeks ago.
Agreed that we should nickname the test the anti-Colin Robinson test.

>
